File: python/prob_1044.py
# Lendo dois valores inteiros
a, b = map(int, input().split())

"""
Verificando se eles são multiplos:
Caso o resto da divisão do maior pelo menor for 0, 
então eles são multiplos, pois para serem multiplos 
a = b * k, sendo k um número inteiro
"""
if a >= b:
    if a % b == 0:
        print("Sao Multiplos")
    else:
        print("Nao sao Multiplos")
else:
    if b % a == 0:
        print("Sao Multiplos")
    else:
        print("Nao sao Multiplos")

File: java/prob_1044.java
import java.util.Scanner;

public class prob_1044 {
    public static void main(String[] args){
        int a, b, resto;
        Scanner sc = new Scanner(System.in);

        a = sc.nextInt();
        b = sc.nextInt();

        if (a > b)
            resto = a % b;
        else
            resto = b % a;

        if (resto == 0)
            System.out.println("Sao Multiplos");
        else
            System.out.println("Nao sao Multiplos");
        sc.close();
    }
}
